psst-gui/src/ui/mod.rs

```diff
@@ -150,18 +149,18 @@ fn root_widget() -> impl Widget<AppState> {
         .with_flex_child(Overlay::bottom(route_widget(), alert_widget()), 1.0)
         .with_child(playback::panel_widget())
         .background(theme::BACKGROUND_LIGHT);
-
-    let split = Split::columns(sidebar, main)
-        .split_point(0.2)
-        .bar_size(1.0)
-        .min_size(150.0, 300.0)
-        .min_bar_area(1.0)
-        .solid_bar(true);
```

Comment:
I'm wondering if maybe if it wouldn't be worthwhile trying to use two Split:columns() one for `let split = Split::columns(sidebar, main)` and then one for the `Split::columns(split, queue_widget)`? Maybe this would allow the split borders to exist nicely? Just a thought

Reply:
This would only work for the left and not the queue widget. This is because it will still leave remaining space for the widget even if it is empty.

But it would work to stop the overlapping on the left which isn't an issue in the right so this would work if we only use the split widget for the left sidebar. 

Does that make sense?